cmd/krel/cmd/changelog.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// changelogCmd represents the subcommand for `krel changelog`
+var changelogCmd = &cobra.Command{
+	Use:   "changelog",
+	Short: "changelog maintains the lifecycle of CHANGELOG-x.y.md files",
+	Long: strings.Join([]string{
```

Comment:
Can we just do a multi-line string here?
e.g., https://github.com/kubernetes/release/blob/1d3b96dcbff440fbd3c4653cca64f116b8168acb/cmd/krel/cmd/ff.go#L42-L45

Reply:
Sure, I made it a multi-line raw string.